server/channels/store/store.go

```diff
@@ -1211,9 +1211,8 @@ type PostReminderMetadata struct {
 // SidebarCategorySearchOpts contains the options for a graphQL query
 // to get the sidebar categories.
 type SidebarCategorySearchOpts struct {
```

Comment:
Do we even need this type any more or could we go back to just passing around the team ID as a string? 

If you want to keep it around, I'd also be good with that, although we should probably still remove the `Type` field then since it doesn't appear to be used

Reply:
Yes, good suggestion. Done!